src/main/java/project_16x16/Util.java

```diff
@@ -104,6 +104,7 @@ public static PImage scale(PImage pBuffer, int scaling) {
 	public static PImage resizeImage(PImage img, float scl) {
 		PGraphics pg = applet.createGraphics((int) (img.width * scl), (int) (img.height * scl));
 
+		pg.noSmooth();
```

Comment:
Does this actually work? I have a memory that it doesn't (bicubic scaling instead).

Reply:
Not really sure what you mean by bicubic scaling, but I don't see any problems with it.

Reply:
As in the PGraphics object doesn't support noSmooth(), but I'm not 100% sure.